nucypher/crypto/powers.py

```diff
@@ -209,10 +202,30 @@ class EncryptingPower(KeyPairBasedPower):
     provides = ("decrypt",)
 
 
+class DerivedKeyBasedPower(CryptoPowerUp):
+    """
+    Rather than rely on an established KeyPair, this type of power
+    derives a key at moments defined by the user.
+    """
+    pass
```

Comment:
No need to pass here; docstring is enough.

Reply:
✅